### PR TITLE
[BEAM-13395] [Playground] Tag katas

### PR DESCRIPTION
**Pull request by pavel-avilov**
[[BEAM-13395]](https://issues.apache.org/jira/browse/BEAM-13395)
Tag java and python katas

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

`ValidatesRunner` compliance status (on master branch)
--------------------------------------------------------

<table>
  <thead>
    <tr>
      <th>Lang</th>
      <th>ULR</th>
      <th>Dataflow</th>
      <th>Flink</th>
      <th>Samza</th>
      <th>Spark</th>
      <th>Twister2</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Go</td>
      <td>---</td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/">
          Build Status
        </a>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Go_VR_Flink/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Go_VR_Samza/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Go_VR_Spark/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>---</td>
    </tr>
    <tr>
      <td>Java</td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_ULR/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Streaming/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Java11/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_VR_Dataflow_V2/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_VR_Dataflow_V2_Streaming/lastCompletedBuild/">
          Build Status
        </a><br>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Java11/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_PVR_Samza/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_PVR_Spark_Batch/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_SparkStructuredStreaming/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Twister2/lastCompletedBuild/">
          Build Status
        </a>
      </td>
    </tr>
    <tr>
      <td>Python</td>
      <td>---</td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Py_VR_Dataflow_V2/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Python_PVR_Flink_Cron/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Python_VR_Flink/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Python_VR_Samza/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Python_VR_Spark/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>---</td>
    </tr>
    <tr>
      <td>XLang</td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_XVR_Direct/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_XVR_Dataflow/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_XVR_Flink/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_XVR_Samza/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_XVR_Spark/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>---</td>
    </tr>
  </tbody>
</table>

Examples testing status on various runners
--------------------------------------------------------

<table>
  <thead>
    <tr>
      <th>Lang</th>
      <th>ULR</th>
      <th>Dataflow</th>
      <th>Flink</th>
      <th>Samza</th>
      <th>Spark</th>
      <th>Twister2</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Go</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
    </tr>
    <tr>
      <td>Java</td>
      <td>---</td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Java_Examples_Dataflow_Cron/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Java_Examples_Dataflow_Java11_Cron/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java_Examples_Dataflow_V2/lastCompletedBuild/">
          Build Status
        </a><br>
      </td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
    </tr>
    <tr>
      <td>Python</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
    </tr>
    <tr>
      <td>XLang</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
    </tr>
  </tbody>
</table>

Post-Commit SDK/Transform Integration Tests Status (on master branch)
------------------------------------------------------------------------------------------------

<table>
  <thead>
    <tr>
      <th>Go</th>
      <th>Java</th>
      <th>Python</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Python36/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Python37/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PostCommit_Python38/lastCompletedBuild/">
          Build Status
        </a>
      </td>
    </tr>
  </tbody>
</table>

Pre-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

<table>
  <thead>
    <tr>
      <th>---</th>
      <th>Java</th>
      <th>Python</th>
      <th>Go</th>
      <th>Website</th>
      <th>Whitespace</th>
      <th>Typescript</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Non-portable</td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Java_Cron/lastCompletedBuild/">
          Build Status
        </a><br>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Python_Cron/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_PythonLint_Cron/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_PythonDocker_Cron/lastCompletedBuild/">
          Build Status
        </a><br>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_PythonDocs_Cron/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Go_Cron/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Website_Cron/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Whitespace_Cron/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Typescript_Cron/lastCompletedBuild/">
          Build Status
        </a>
      </td>
    </tr>
    <tr>
      <td>Portable</td>
      <td>---</td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_Portable_Python_Cron/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>
        <a href="https://ci-beam.apache.org/job/beam_PreCommit_GoPortable_Cron/lastCompletedBuild/">
          Build Status
        </a>
      </td>
      <td>---</td>
      <td>---</td>
      <td>---</td>
    </tr>
  </tbody>
</table>

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by pavel-avilov** [comment]
R: @KhaninArtur @ilya-kozyrev @daria-malkova @AydarZaynutdinov

File: learning/katas/java/Core Transforms/Combine/BinaryCombineFn Lambda/src/org/apache/beam/learning/katas/coretransforms/combine/binarycombinefnlambda/Task.java [hunk 1]
```diff
@@ -18,6 +18,15 @@
 
 package org.apache.beam.learning.katas.coretransforms.combine.binarycombinefnlambda;
 
+// beam-playground:
+//   name: BinaryCombineFnLambda
+//   description: Task from katas to implements the summation of BigInteger using lambda or method reference.
```
Comment (daria-malkova) [inline learning/katas/java/Core Transforms/Combine/BinaryCombineFn Lambda/src/org/apache/beam/learning/katas/coretransforms/combine/binarycombinefnlambda/Task.java:23]: just lambda

**Review by daria-malkova — CHANGES_REQUESTED** [review]

File: learning/katas/java/Core Transforms/Side Output/Side Output/src/org/apache/beam/learning/katas/coretransforms/sideoutput/Task.java [hunk 2]
```diff
@@ -18,6 +18,15 @@
...
 package org.apache.beam.learning.katas.coretransforms.sideoutput;
 
+// beam-playground:
+//   name: SideOutput
+//   description: Task from katas to implement additional output to your ParDo for numbers bigger than 100.
+//   multifile: false
+//   pipeline_options:
+//   categories:
+//     - Combiners
+//     - Testing
```
Comment (KhaninArtur) [inline learning/katas/java/Core Transforms/Side Output/Side Output/src/org/apache/beam/learning/katas/coretransforms/sideoutput/Task.java:28]:
```suggestion
//     - Testing
//     - Multiple Outputs
```

File: learning/katas/java/Windowing/Fixed Time Window/Fixed Time Window/src/org/apache/beam/learning/katas/windowing/fixedwindow/Task.java [hunk 3]
```diff
@@ -18,6 +18,14 @@
 
 package org.apache.beam.learning.katas.windowing.fixedwindow;
 
+// beam-playground:
+//   name: FixedTimeWindow
+//   description: Task from katas to count the number of events that happened based on fixed window with 1-day duration.
+//   multifile: false
+//   pipeline_options:
+//   categories:
+//     - Testing
```
Comment (KhaninArtur) [inline learning/katas/java/Windowing/Fixed Time Window/Fixed Time Window/src/org/apache/beam/learning/katas/windowing/fixedwindow/Task.java:27]:
```suggestion
//     - Combiners
//     - Testing
```

File: learning/katas/python/Core Transforms/Side Output/Side Output/task.py [hunk 4]
```diff
@@ -14,6 +14,15 @@
...
 #  limitations under the License.
 
+# beam-playground:
+#   name: SideOutput
+#   description: Task from katas to implement additional output to your ParDo for numbers bigger than 100.
+#   multifile: false
+#   pipeline_options:
+#   categories:
+#     - Combiners
+#     - Testing
```
Comment (KhaninArtur) [inline learning/katas/python/Core Transforms/Side Output/Side Output/task.py:24]:
```suggestion
#     - Testing
#     - Multiple Outputs
```

**Review by KhaninArtur — CHANGES_REQUESTED** [review]

File: learning/katas/python/Core Transforms/Side Output/Side Output/task.py (hunk 4 above)
Comment (pavel-avilov) [reply learning/katas/python/Core Transforms/Side Output/Side Output/task.py:24]: Done

File: learning/katas/java/Windowing/Fixed Time Window/Fixed Time Window/src/org/apache/beam/learning/katas/windowing/fixedwindow/Task.java (hunk 3 above)
Comment (pavel-avilov) [reply learning/katas/java/Windowing/Fixed Time Window/Fixed Time Window/src/org/apache/beam/learning/katas/windowing/fixedwindow/Task.java:27]: Done

File: learning/katas/java/Core Transforms/Side Output/Side Output/src/org/apache/beam/learning/katas/coretransforms/sideoutput/Task.java (hunk 2 above)
Comment (pavel-avilov) [reply learning/katas/java/Core Transforms/Side Output/Side Output/src/org/apache/beam/learning/katas/coretransforms/sideoutput/Task.java:28]: Done

File: learning/katas/java/Core Transforms/Combine/BinaryCombineFn Lambda/src/org/apache/beam/learning/katas/coretransforms/combine/binarycombinefnlambda/Task.java (hunk 1 above)
Comment (pavel-avilov) [reply learning/katas/java/Core Transforms/Combine/BinaryCombineFn Lambda/src/org/apache/beam/learning/katas/coretransforms/combine/binarycombinefnlambda/Task.java:23]: Done

File: learning/katas/java/Core Transforms/Map/FlatMapElements/src/org/apache/beam/learning/katas/coretransforms/map/flatmapelements/Task.java [hunk 5]
```diff
@@ -21,7 +21,7 @@
 //  eam-playground:
 //   name: FlatMap
 //   description: Task from katas to implement a function that maps each input sentence
-//       into words tokenized by whitespace (" ").
+//       into splitted whitespace (" ").
```
Comment (KhaninArtur) [inline learning/katas/java/Core Transforms/Map/FlatMapElements/src/org/apache/beam/learning/katas/coretransforms/map/flatmapelements/Task.java:24]:
```suggestion
//       into words splitted by whitespace (" ").
```

File: learning/katas/java/Core Transforms/Map/ParDo OneToMany/src/org/apache/beam/learning/katas/coretransforms/map/pardoonetomany/Task.java [hunk 6]
```diff
@@ -20,7 +20,7 @@
 
 // beam-playground:
 //   name: MapParDoOneToMany
-//   description: Task from katas that maps each input sentence into words tokenized by whitespace (" ").
+//   description: Task from katas that maps each input sentence into splitted whitespace (" ").
```
Comment (KhaninArtur) [inline learning/katas/java/Core Transforms/Map/ParDo OneToMany/src/org/apache/beam/learning/katas/coretransforms/map/pardoonetomany/Task.java:23]:
```suggestion
//   description: Task from katas that maps each input sentence into words splitted by whitespace (" ").
```

File: learning/katas/python/Core Transforms/Map/FlatMap/task.py [hunk 7]
```diff
@@ -17,7 +17,7 @@
 # beam-playground:
 #   name: FlatMap
 #   description: Task from katas to implement a function that maps each input sentence
-#       into words tokenized by whitespace (" ").
+#       into splitted whitespace (" ").
```
Comment (KhaninArtur) [inline learning/katas/python/Core Transforms/Map/FlatMap/task.py:20]:
```suggestion
#       into words splitted by whitespace (" ").
```

File: learning/katas/python/Core Transforms/Map/ParDo OneToMany/task.py [hunk 8]
```diff
@@ -16,7 +16,7 @@
 
 # beam-playground:
 #   name: MapParDoOneToMany
-#   description: Task from katas is a ParDo that maps each input sentence into words tokenized by whitespace (" ").
+#   description: Task from katas is a ParDo that maps each input sentence into splitted whitespace (" ").
```
Comment (KhaninArtur) [inline learning/katas/python/Core Transforms/Map/ParDo OneToMany/task.py:19]:
```suggestion
#   description: Task from katas is a ParDo that maps each input sentence into words splitter by whitespace (" ").
```

**Review by KhaninArtur — CHANGES_REQUESTED** [review]

File: learning/katas/java/Core Transforms/Map/FlatMapElements/src/org/apache/beam/learning/katas/coretransforms/map/flatmapelements/Task.java (hunk 5 above)
Comment (pavel-avilov) [reply learning/katas/java/Core Transforms/Map/FlatMapElements/src/org/apache/beam/learning/katas/coretransforms/map/flatmapelements/Task.java:24]: Done.

File: learning/katas/java/Core Transforms/Map/ParDo OneToMany/src/org/apache/beam/learning/katas/coretransforms/map/pardoonetomany/Task.java (hunk 6 above)
Comment (pavel-avilov) [reply learning/katas/java/Core Transforms/Map/ParDo OneToMany/src/org/apache/beam/learning/katas/coretransforms/map/pardoonetomany/Task.java:23]: Done.

File: learning/katas/python/Core Transforms/Map/FlatMap/task.py (hunk 7 above)
Comment (pavel-avilov) [reply learning/katas/python/Core Transforms/Map/FlatMap/task.py:20]: Done.

**Review by KhaninArtur — APPROVED** [review]
LGTM

File: learning/katas/python/Common Transforms/Filter/ParDo/task.py [hunk 9]
```diff
@@ -14,6 +14,15 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
+# beam-playground:
+#   name: FilterParDo
+#   description: Task from katas to implement a filter function that filters out the even numbers.
+#   multifile: false
+#   pipeline_options:
+#   categories:
+#     - Combiners
```
Comment (ilya-kozyrev) [inline learning/katas/python/Common Transforms/Filter/ParDo/task.py:23]:
```suggestion
#     - Filtering
```

File: learning/katas/python/Core Transforms/Branching/Branching/task.py [hunk 10]
```diff
@@ -14,6 +14,16 @@
...
 #  limitations under the License.
 
+# beam-playground:
+#   name: Branching
+#   description: Task from katas to branch out the numbers to two different transforms, one transform
+#     is multiplying each number by 5 and the other transform is multiplying each number by 10.
+#   multifile: false
+#   pipeline_options:
+#   categories:
+#     - Combiners
```
Comment (ilya-kozyrev) [inline learning/katas/python/Core Transforms/Branching/Branching/task.py:24]:
```suggestion
#     - Branching
#     - Multiple Outputs
```

File: learning/katas/python/Core Transforms/Composite Transform/Composite Transform/task.py [hunk 11]
```diff
@@ -14,6 +14,16 @@
...
 #  limitations under the License.
 
+# beam-playground:
+#   name: CompositeTransform
+#   description: Task from katas to implement a composite transform "ExtractAndMultiplyNumbers"
+#     that extracts numbers from comma separated line and then multiplies each number by 10.
+#   multifile: false
+#   pipeline_options:
+#   categories:
+#     - Combiners
```
Comment (ilya-kozyrev) [inline learning/katas/python/Core Transforms/Composite Transform/Composite Transform/task.py:24]:
```suggestion
#     - Flatten
```

File: learning/katas/python/Core Transforms/Flatten/Flatten/task.py [hunk 12]
```diff
@@ -14,6 +14,15 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
+# beam-playground:
+#   name: Flatten
+#   description: Task from katas that merges two PCollections of words into a single PCollection.
+#   multifile: false
+#   pipeline_options:
+#   categories:
+#     - Combiners
```
Comment (ilya-kozyrev) [inline learning/katas/python/Core Transforms/Flatten/Flatten/task.py:23]:
```suggestion
#     - Flatten
```

File: learning/katas/python/Core Transforms/Map/FlatMap/task.py [hunk 13]
```diff
@@ -14,6 +14,16 @@
...
 #   limitations under the License.
 
+# beam-playground:
+#   name: FlatMap
+#   description: Task from katas to implement a function that maps each input sentence
+#     into words split by whitespace (" ").
+#   multifile: false
+#   pipeline_options:
+#   categories:
+#     - Combiners
```
Comment (ilya-kozyrev) [inline learning/katas/python/Core Transforms/Map/FlatMap/task.py:24]:
```suggestion
#     - Core Transforms
#     - Flatten
```

File: learning/katas/python/Core Transforms/Map/Map/task.py [hunk 14]
```diff
@@ -14,6 +14,15 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
+# beam-playground:
+#   name: Map
+#   description: Task from katas to implement a simple map function that multiplies all input elements by 5.
+#   multifile: false
+#   pipeline_options:
+#   categories:
+#     - Combiners
```
Comment (ilya-kozyrev) [inline learning/katas/python/Core Transforms/Map/Map/task.py:23]:
```suggestion
#     - Core Transforms
```

File: learning/katas/python/Core Transforms/Map/ParDo OneToMany/task.py [hunk 15]
```diff
@@ -14,6 +14,16 @@
...
 #   limitations under the License.
 
+# beam-playground:
+#   name: MapParDoOneToMany
+#   description: Task from katas is a ParDo that maps each input sentence into
+#     words splitter by whitespace (" ").
+#   multifile: false
+#   pipeline_options:
+#   categories:
+#     - Combiners
```
Comment (ilya-kozyrev) [inline learning/katas/python/Core Transforms/Map/ParDo OneToMany/task.py:24]:
```suggestion
#     - Core Transforms
#     - Flatten
```

File: learning/katas/python/Core Transforms/Map/ParDo/task.py [hunk 16]
```diff
@@ -14,6 +14,15 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
+# beam-playground:
+#   name: MapPardo
+#   description: Task from katas is simple ParDo that maps the input element by multiplying it by 10.
+#   multifile: false
+#   pipeline_options:
+#   categories:
+#     - Combiners
```
Comment (ilya-kozyrev) [inline learning/katas/python/Core Transforms/Map/ParDo/task.py:23]:
```suggestion
#     - Core Transforms
```

File: learning/katas/python/Core Transforms/Partition/Partition/task.py [hunk 17]
```diff
@@ -14,6 +14,16 @@
...
 #   limitations under the License.
 
+# beam-playground:
+#   name: Partition
+#   description: Task from katas that splits a PCollection of numbers into two PCollections. The first PCollection
+#     contains numbers greater than 100, and the second PCollection contains the remaining numbers.
+#   multifile: false
+#   pipeline_options:
+#   categories:
+#     - Combiners
```
Comment (ilya-kozyrev) [inline learning/katas/python/Core Transforms/Partition/Partition/task.py:24]:
```suggestion
#     - Multiple Outputs
```

File: learning/katas/python/Core Transforms/Side Input/Side Input/task.py [hunk 18]
```diff
@@ -22,6 +22,16 @@
...
 #  limitations under the License.
 
+# beam-playground:
+#   name: SideInput
+#   description: Task from katas to enrich each Person with the country based on the city he/she lives in.
+#   multifile: false
+#   pipeline_options:
+#   categories:
+#     - Side Input
+#     - Combiners
```
Comment (ilya-kozyrev) [inline learning/katas/python/Core Transforms/Side Input/Side Input/task.py:32]:
```suggestion

```

File: learning/katas/python/Core Transforms/Side Output/Side Output/task.py [hunk 19]
```diff
@@ -14,6 +14,16 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+# beam-playground:
+#   name: SideOutput
+#   description: Task from katas to implement additional output to your ParDo for numbers bigger than 100.
+#   multifile: false
+#   pipeline_options:
+#   categories:
+#     - Combiners
```
Comment (ilya-kozyrev) [inline learning/katas/python/Core Transforms/Side Output/Side Output/task.py:23]:
```suggestion
#     - Filtering
```

File: learning/katas/python/Windowing/Adding Timestamp/ParDo/task.py [hunk 20]
```diff
@@ -14,6 +14,15 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
+# beam-playground:
+#   name: WindowingParDo
+#   description: Task from katas to assign each element a timestamp based on the the `Event.timestamp`.
+#   multifile: false
+#   pipeline_options:
+#   categories:
+#     - Combiners
```
Comment (ilya-kozyrev) [inline learning/katas/python/Windowing/Adding Timestamp/ParDo/task.py:23]:
```suggestion
#     - Streaming
```

File: learning/katas/python/Windowing/Fixed Time Window/Fixed Time Window/task.py [hunk 21]
```diff
@@ -14,6 +14,15 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
+# beam-playground:
+#   name: WindowingFixedTime
+#   description: Task from katas to count the number of events that happened based on fixed window with 1-day duration.
+#   multifile: false
+#   pipeline_options:
+#   categories:
+#     - Combiners
```
Comment (ilya-kozyrev) [inline learning/katas/python/Windowing/Fixed Time Window/Fixed Time Window/task.py:23]:
```suggestion
#     - Streaming
```

File: playground/categories.yaml [hunk 22]
```diff
@@ -30,3 +30,4 @@ categories:
   - Options
   - Coders
   - Stateful Processing
+  - Beam SQL
```
Comment (ilya-kozyrev) [inline playground/categories.yaml:33]:
```suggestion
  - Beam SQL
  - Filtering
  - Branching
  - Flatten
  - Core Transforms
```

**Review by ilya-kozyrev — CHANGES_REQUESTED** [review]
Just want to add new categories and reduce combiners

**Comment by pavel-avilov** [comment]
@ilya-kozyrev I added all your suggested changes

**Review by KhaninArtur — APPROVED** [review]

**Review by ilya-kozyrev — APPROVED** [review]
LGTM

**Review by daria-malkova — APPROVED** [review]
LGTM

**Comment by pavel-avilov** [comment]
R: @pabloem

File: playground/categories.yaml [hunk 23]
```diff
@@ -30,3 +30,12 @@ categories:
...
+  - Filtering
+  - Branching
+  - Flatten
+  - Core Transforms
+  - Filtering
+  - Branching
+  - Flatten
+  - Core Transforms
```
Comment (pabloem) [inline playground/categories.yaml:41]: these lines seem to be repeated

File: learning/katas/python/Windowing/Fixed Time Window/Fixed Time Window/task.py [hunk 24]
```diff
@@ -14,6 +14,14 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
+# beam-playground:
+#   name: WindowingFixedTime
+#   description: Task from katas to count the number of events that happened based on fixed window with 1-day duration.
+#   multifile: false
+#   pipeline_options:
+#   categories:
+#     - Streaming
```
Comment (pabloem) [inline learning/katas/python/Windowing/Fixed Time Window/Fixed Time Window/task.py:23]:
```suggestion
#     - Streaming
#     - Windowing
#     - Combiners
```

Comment (pavel-avilov) [reply learning/katas/python/Windowing/Fixed Time Window/Fixed Time Window/task.py]: I agree, done.

File: playground/categories.yaml (hunk 23 above)
Comment (pavel-avilov) [reply playground/categories.yaml]: Done.

**Comment by pabloem** [comment]
LGTM

**Comment by iht** [comment]
The answers to the katas for the Edu plugin of PyCharm/IntelliJ are now in wrong locations, and the katas are not working. Please update also the placeholders when modifying the text of the katas. I am working on a fix to this problem ([BEAM-13836](https://issues.apache.org/jira/browse/BEAM-13836).

**Pull request merged**